VideoLocker/res/layout/fragment_course_detail.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:background="@color/edx_grayscale_neutral_white"
+    android:orientation="vertical"
+    android:paddingLeft="20dp"
+    android:paddingRight="20dp"
+
+    >
+
+    <org.edx.mobile.view.custom.ETextView
+        android:id="@+id/course_detail_short_description"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:gravity="left"
+        android:paddingTop="15dp"
+        android:textColor="@color/edx_grayscale_neutral_x_dark"
+        android:textSize="@dimen/edx_base"
+        tools:text="Bacon ipsum dolor amet bresaola bacon shoulder chicken beef ribs rump andouille turkey pig frankfurter tail sausage leberkas fatback."/>
+
+    <org.edx.mobile.view.custom.EButton
+        android:id="@+id/button_enroll_now"
+        style="@style/edX.Widget.EnrollButton"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="15dp"
+        android:onClick="enrollButtonClicked"
+        android:padding="10dp" />
+
+    <LinearLayout
+        android:id="@+id/course_detail_fields"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"></LinearLayout>
+
+    <LinearLayout
+        android:id="@+id/course_detail_course_about"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:orientation="vertical">
+
+        <org.edx.mobile.view.custom.ETextView
+            android:id="@+id/about_course_text"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:paddingTop="15dp"
+            android:text="@string/about_this_course"
+            style="@style/bold_regular_edx_black_text"
+            android:textSize="@dimen/edx_x_large" />
+
+        <RelativeLayout
```

Comment:
nit: You aren't actually using any features of `RelativeLayout`, so this could be changed to a simple `FrameLayout`